ui/src/components/Icon.tsx

```diff
@@ -659,6 +666,67 @@ export function Icon({ color = 'var(--black)', name, onClick, style = {} }: Prop
           </defs>
         </svg>
       )
+    case 'bubble-chat':
+      return (
+        <svg width="56" height="56" viewBox="0 0 56 56" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path
+            d="M27.9768 1.00005C23.2092 0.990842 18.5245 2.249 14.4018 4.64603C10.2792 7.04305 6.86623 10.493 4.51187 14.6431C2.15752 18.7931 0.946302 23.4946 1.00183 28.2668C1.05735 33.0391 2.37768 37.7112 4.82795 41.8052L1.40365 54.6009L14.186 51.173C17.7536 53.3086 21.7659 54.5898 25.9098 54.9166C30.0538 55.2434 34.2171 54.6069 38.0748 53.057C41.9324 51.5071 45.3799 49.0855 48.1482 45.9814C50.9166 42.8774 52.9308 39.1749 54.0336 35.1629C55.1363 31.1509 55.2978 26.9381 54.5054 22.8533C53.713 18.7685 51.9884 14.9221 49.4658 11.6148C46.9432 8.30754 43.6911 5.62871 39.9635 3.78737C36.2358 1.94604 32.1335 0.992118 27.9768 1.00005Z"
+            stroke="#555ABF"
+            stroke-width="2"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          />
+          <path
+            d="M15.0767 22.1304H40.8758"
+            stroke="#555ABF"
+            stroke-width="2"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          />
+          <path
+            d="M15.0767 29.1738H40.8758"
+            stroke="#555ABF"
+            stroke-width="2"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          />
+          <path
+            d="M15.0767 36.2175H40.8758"
+            stroke="#555ABF"
+            stroke-width="2"
+            stroke-linecap="round"
+            stroke-linejoin="round"
```

Comment:
since this is JSX these will all throw errors for being kebab-case. they need to be camelCase